docs/sequences.md

```diff
@@ -1,4 +1,4 @@
-# Sequences
+# Sequences/Macros
```

Comment:
I very intentionally strayed away from calling these "macros". It's a very loaded term that carries way too many expectations with it.

Not redlining this, more opening a discussion around word choice here. I'm slowly trying to establish a bit of a "voice" for our docs, and consistency is part of that.

Reply:
I left it as secondary as people will eventually ask "how do I make a macro like *other keyboard*". Calling them sequences may be correct, but not at least mentioning the term that people refer to this functionality by most commonly seems foolish to me, and begging for support threads/messages asking "Why ain't I got no macros?"

Reply:
I believe that I have come up with a compromise. Please mark this as resolved if that meets your requirements.